### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by af-a**
This pull request updates the documentation of this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ In order to deal with data that come from different sources (e.g. ROS, EtherCAT,
 Data from the black box can be retrieved through a Zyre-based query interface that listens to JSON messages and responds back with JSON messages. The following message types can be sent to the query interface:
 * variable query: returns a list of all variables logged on a particular black box (schema defined in [docs/messages/black-box-variable-query-schema.json](docs/messages/black-box-variable-query-schema.json))
 * data query: returns data corresponding to a set of variables in a given time interval (schema defined in [docs/messages/black-box-data-query-schema.json](docs/messages/black-box-data-query-schema.json))
+* latest data query: returns the latest datum corresponding to a set of variables
```
Comment (alex-mitrevski) [inline README.md]: We need to include a schema for this message, just like for the other message types.

File: README.md [hunk 2]
```diff
@@ -133,6 +134,8 @@ Both Python components (the logger and the query interface) depend on the follow
 * The [`Pyre base communicator`](https://github.com/ropod-project/ropod_common/tree/master/pyropod/ropod/pyre_communicator) in `ropod_common`
 * `PyYAML`
 * `pymongo`
+* `rosnode`
```
Comment (alex-mitrevski) [inline README.md]: `rosnode` is not a package. You can add `rospkg`, and perhaps `catkin-pkg` as well.

File: README.md [hunk 3]
```diff
@@ -133,6 +134,8 @@ Both Python components (the logger and the query interface) depend on the follow
...
+* `zmq`
```
Comment (alex-mitrevski) [inline README.md]: This would be `pyzmq`

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]
Could you please also add the new dependencies to the [`requirements.txt`](https://github.com/ropod-project/black-box/blob/master/pybb/requirements.txt) file in `pybb`?

File: README.md (hunk 1 above)
Comment (af-a) [reply README.md]: Resolved [here](https://github.com/ropod-project/black-box/pull/2/commits/66486c0b4fd273d89b7b242160daa2fde277bea8).

File: README.md (hunk 2 above)
Comment (af-a) [reply README.md]: Resolved [here](https://github.com/ropod-project/black-box/pull/2/commits/f85ad00eb2f41e0a591f74f7f121760bdfce2ffc).

File: README.md (hunk 3 above)
Comment (af-a) [reply README.md]: Resolved [here](https://github.com/ropod-project/black-box/pull/2/commits/f85ad00eb2f41e0a591f74f7f121760bdfce2ffc).

**Review by alex-mitrevski — APPROVED** [review]

**Pull request merged**